docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -411,15 +302,110 @@ Payload:
 }
 ```
 
-## Skip organization role sync
+## Examples of setting up generic OAuth2
 
-To prevent the sync of organization roles from the OAuth provider, set `skip_org_role_sync` to `true`. This is useful if you want to manage the organization roles for your users from within Grafana.
-This also impacts the `allow_assign_grafana_admin` setting by not syncing the Grafana admin role from the OAuth provider.
+This section includes examples of setting up generic OAuth2 integration.
 
-```ini
-[auth.generic_oauth]
-# ..
-# prevents the sync of org roles from the Oauth provider
-skip_org_role_sync = true
-``
-```
+### Set up OAuth2 with Auth0
+
+To set up generic OAuth2 authentication with Auth0, follow these steps:
+
+1. Create an application using the following parameters:
+
+   - Name: Grafana
+   - Type: Regular Web Application
+
+1. Go to the **Settings** tab and set **Allowed Callback URLs** to `https://<grafana domain>/login/generic_oauth`.
+
+1. Click **Save Changes**.
+
+1. Use the values from the **Settings** tab to configure Grafana:
+
+   ```bash
+   [auth.generic_oauth]
+   enabled = true
+   allow_sign_up = true
+   auto_login = false
+   team_ids =
+   allowed_organizations =
+   name = Auth0
+   client_id = <client id>
+   client_secret = <client secret>
+   scopes = openid profile email offline_access
+   auth_url = https://<domain>/authorize
+   token_url = https://<domain>/oauth/token
+   api_url = https://<domain>/userinfo
+   use_pkce = true
+   ```
+
+### Set up OAuth2 with Bitbucket
+
+To set up generic OAuth2 authentication with Bitbucket, follow these steps:
+
+1. Navigate to **Settings > Workspace setting > OAuth consumers**.
+
+1. Create an application by selecting **Add consumer** and using the following parameters:
+
+   - Allowed Callback URLs: `https://<grafana domain>/login/generic_oauth`
+
+1. Click **Save**.
+
+1. Use the `Key` and `Secret` from the consumer description to configure Grafana:
```

Comment:
Update this step based on earlier comment about where and how they do this in Grafana.